dev/com.ibm.ws.org.eclipse.yasson.1.0/bnd.bnd

```diff
@@ -28,3 +28,5 @@ instrument.disabled: true
     org.hamcrest:hamcrest-all;version=1.3, \
     org.jmock:jmock-junit4;strategy=exact;version=2.5.1, \
     org.jmock:jmock;strategy=exact;version=2.5.1
+    
+jakartaeeMe: true
```

Comment:
what does this do?

Reply:
jakartaeeMe causes the build to create the jakartaee-enabled version of the yasson-1.0 bundle, which is used by feature jsonbImpl-2.0.0.